File: prog/c/deleteelement.c
#include<stdio.h>
#include<stdlib.h>
int main()
{
	int *a,n,i,element;
	printf("Enter size of array:");
	scanf("%d",&n);

	a=malloc(sizeof(int)*n);

	printf("Enter %d Elements:",n);
	for(i=0;i<n;i++)
	{
		scanf("%d",&a[i]);
	}
	printf("Enter a Element to delete:");
	scanf("%d",&element);
	for(i=0;i<n;i++)
	{
		if(a[i]==element)
		{
			while(i<n-1)	
			{
				a[i]=a[i+1];
				i++;
			}
			a[i]=0;
		}
	}
	printf("After deleting %d:",element);
	for(i=0;i<n;i++)
	{
		printf("%d ",a[i]);
	}
	return 0;
				
}

File: prog/python/deleteelement.py
l=list(map(int,input("Enter Numbers:").split()))
e=int(input("Enter the element to delete"))
for i in range(0,len(l)):
    if(l[i]==e):
        for j in range(i,len(l)-1):
            l[j]=l[j+1]
        l[j+1]=None
print(l)
            
